multicloud-bootstrap/init.sh

```diff
@@ -0,0 +1,296 @@
+#!/bin/bash
+#
+# This is the init script that will call the individual Cloud specific script
+#
+
+## Inputs
+export CLOUD_TYPE=$1
+export OFFERING_TYPE=$2
+export DEPLOY_REGION=$3
+export ACCOUNT_ID=$4
+export CLUSTER_SIZE=$5
+export RANDOM_STR=$6
+export BASE_DOMAIN=$7
+export SSH_KEY_NAME=$8
+export DEPLOY_WAIT_HANDLE=$9
+export SLS_ENTITLEMENT_KEY=${10}
+export OCP_PULL_SECRET=${11}
+export MAS_LICENSE_URL=${12}
+export SLS_ENDPOINT_URL=${13}
+export SLS_REGISTRATION_KEY=${14}
+export SLS_PUB_CERT_URL=${15}
+export BAS_ENDPOINT_URL=${16}
+export BAS_API_KEY=${17}
+export BAS_PUB_CERT_URL=${18}
+export MAS_JDBC_USER=${19}
+export MAS_JDBC_PASSWORD=${20}
+export MAS_JDBC_URL=${21}
+export MAS_JDBC_CERT_URL=${22}
+export MAS_DB_IMPORT_DEMO_DATA=${23}
+export EXS_OCP_URL=${24}
+export EXS_OCP_USER=${25}
+export EXS_OCP_PWD=${26}
+
+# Load helper functions
+. helper.sh
+export -f log
+export -f get_mas_creds
+export -f retrieve_mas_ca_cert
+export -f mark_provisioning_failed
+export -f get_sls_endpoint_url
+export -f get_sls_registration_key
+export -f get_bas_endpoint_url
+export -f get_bas_api_key
+
+# Check for input parameters
+if [[ (-z $CLOUD_TYPE) || (-z $DEPLOY_REGION) || (-z $ACCOUNT_ID) || (-z $CLUSTER_SIZE) \
+   || (-z $RANDOM_STR) || (-z $BASE_DOMAIN) || (-z $SSH_KEY_NAME) || (-z $DEPLOY_WAIT_HANDLE) ]]; then
+  log "ERROR: Required parameter not specified, please provide all the required inputs to the script."
+  PRE_VALIDATION=fail
+fi
+
+if [[ $OFFERING_TYPE == "MAS Core + Cloud Pak for Data" ]]; then
+  export DEPLOY_CP4D="true"
+  export DEPLOY_MANAGE="false"
+elif [[ $OFFERING_TYPE == "MAS Core + Manage (no Cloud Pak for Data)" ]]; then
+  export DEPLOY_CP4D="false"
+  export DEPLOY_MANAGE="true"
+else
+  log "ERROR: Incorrect value for OFFERING_TYPE - $OFFERING_TYPE"
+  PRE_VALIDATION=fail
+fi
+
+## Variables
+# OCP variables
+export GIT_REPO_HOME=$(pwd)
+export CLUSTER_NAME="masocp-${RANDOM_STR}"
+export OPENSHIFT_USER="masocpuser"
+export OPENSHIFT_PASSWORD=masocp${RANDOM_STR}pass
```

Comment:
can you change these ENV var names to 
OCP_USERNAME  from OPENSHIFT_USER
OCP_PASSWORD from OPENSHIFT_PASSWORD

because as per @durera comment on my PR [Check this comment](https://github.com/ibm-mas/ansible-devops/pull/187#discussion_r825983293) i have modified these env vars

Reply:
Done, I have made the changes